src/sc2laddercore/LadderManager.cpp

```diff
@@ -114,7 +114,7 @@ ExitCase GameUpdate(sc2::Connection *client, sc2::Server *server, const std::str
 	clock_t LastRequest = clock();
 	clock_t FirstRequest = clock();
 	clock_t StepTime = 0;
-	int32_t steps = 0;
+    uint32_t currentGameLoop = 0;
```

Comment:
Note that this file is indented with tabs but this line is indented with spaces.
I'd like to see the conversion to indentation using spaces, but this might not be the right time to address this issue.